kernel/kernel-api/src/main/java/io/delta/kernel/internal/actions/Metadata.java

```diff
@@ -30,6 +30,12 @@
 
 public class Metadata {
 
+  public static Metadata fromRow(Row row) {
+    checkArgument(FULL_SCHEMA.equals(row.getSchema()));
+    return fromColumnVector(
+        VectorUtils.buildColumnVector(Collections.singletonList(row), FULL_SCHEMA), 0);
```

Comment:
0 is a magic number -- can you include the param name at least `, 0 /* paramName */);`